Small update (#45)
SigmaHQ/sigma-specification: Sigma_specification.md

        - [Currently Available Modifiers](#currently-available-modifiers)
          - [Transformations](#transformations)
          - [Types](#types)

>       - [timeframe Search-Identifier](#timeframe-search-identifier)

Reviewer: ```suggestion
      - [Timeframe](#timeframe)
```

  the Elasticsearch query string backend (*es-qs*). Further (like Splunk) are planned or have
  to be implemented by contributors with access to the target systems.

> #### timeframe Search-Identifier

Reviewer: ```suggestion
#### Timeframe
```

> * `timeframe`: It is a special Search-Identifier use **only** with Aggregation condition
>   Defines a time period in which the aggregation should be applied.
>   The following format must be used: number + letter (in lowercase)
>     - Xs seconds
>     - Xm minutes
>     - Xh hours
>     - Xd days
> 
> Example:
> ```yml
>     timeframe: 24h
>     condition: selection | count(dst_port) by src_ip > 10
> ```
> 

Reviewer: ```suggestion
  **Attribute**: timeframe
  
  Is a special Search-Identifier used **only** with Aggregation conditions
  Defines a time period in which the aggregation should be applied.
  The following format must be used: number + letter (in lowercase)
    - Xs seconds
    - Xm minutes
    - Xh hours
    - Xd days

Example:
```yml
    timeframe: 24h
    condition: selection | count(dst_port) by src_ip > 10
```

```

# History

* 2022/12/28 Specification V1.0.2

>   * Add missing `timeframe`

Reviewer: ```suggestion
  * Add missing `timeframe` attribute
```

>   * Add missing `timeframe`

Reviewer: ```suggestion
  * Add missing `timeframe` attribute
```
